sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Bindings/ServiceBusBinding.cs

```diff
@@ -45,8 +45,6 @@ public bool FromAttribute
 
         public async Task<IValueProvider> BindAsync(BindingContext context)
         {
-            context.CancellationToken.ThrowIfCancellationRequested();
```

Comment:
The ValueBindingContext.CancellationToken (which appears to be the same token as BindingContext.CancellationToken) is passed into the TryConvertAsync call on L62. What does that end up doing? Would it still throw? Would be great to have a unit test.